### PR TITLE
Improvement for k8s.io/docs/concepts/overview/working-with-objects/namespaces/  link updated with v1 namespace

### PR DESCRIPTION
**Pull request by hardikshah197**
**What type of PR is this?**
/bug / documentation

**What this PR does / why we need it:**
This PR changes the link in the Phase section of https://kubernetes.io/docs/tasks/administer-cluster/namespaces/ to https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/namespace-v1/
After the changes, the phase section will link to the site that allows details of namespaces.

> A namespace can be in one of two phases:
> 
> Active the namespace is in use
> Terminating the namespace is being deleted, and can not be used for new objects
> See the design doc for more details.

**Which issue(s) this PR fixes:**
**Fixes** #28062 

**_Special notes for your reviewer:_**

**Does this PR introduce a user-facing change?**
fix- Improvement for k8s.io/docs/concepts/overview/working-with-objects/namespaces/
Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

PR is ready for review  /assign @bradtopol i

### CODE REVIEW

**Comment by tengqm** [comment]
I failed to see the improvement from this PR:

- The link target is the API reference, not the design document;
- The API reference provides no more details.

**Comment by hardikshah197** [comment]
Hey @tengqm, Can you please elaborate a little more? Like what exactly you mean with these points:

> The link target is the API reference, not the design document;
> The API reference provides no more details.

Anything will be helpful for me. Thanks

**Review by sftim — DISMISSED** [review]
I agree with @tengqm. The proposed change is not accurate and not useful.

**Comment by hardikshah197** [comment]
> @hardikshah197 would you like some help with how to address #28062?

@sftim sure, any guidance would helpful for me. Thanks

**Comment by hardikshah197** [comment]
>  The link target is the API reference, it is not a design doc, right?

@tengqm Yes, the updated link is not a design doc link.

> In the case of namespace, the API reference is not providing any further details than the docs page

Should I update it with the docs page link? If that would be effective and accurate here.

File: content/en/docs/tasks/administer-cluster/namespaces.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ A namespace can be in one of two phases:
    * `Active` the namespace is in use
    * `Terminating` the namespace is being deleted, and can not be used for new objects
 
-See the [design doc](https://git.k8s.io/community/contributors/design-proposals/architecture/namespaces.md#phases) for more details.
+See the [design doc](https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/namespace-v1/) for more details.
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/namespaces.md:81]:
```suggestion
For more details, see [Namespace](/docs/reference/kubernetes-api/cluster-resources/namespace-v1/)
in the API reference.
```

Comment (sftim) [reply content/en/docs/tasks/administer-cluster/namespaces.md:81]: To make this work: we should amend [k/kubernetes](https://github.com/kubernetes/kubernetes) to mention these details in the doc comments for the Namespace API code:

> A Namespace is in the Active phase if it does not have a ObjectMeta.DeletionTimestamp.
>
> A Namespace is in the Terminating phase if it has a ObjectMeta.DeletionTimestamp.

If accepting this suggestion, we should then /hold this PR until we do that.

Comment (hardikshah197) [reply content/en/docs/tasks/administer-cluster/namespaces.md]: ok, @sftim I got your point, I commit your proposed changes, let's just hold till that Namespace API code being amended. Thanks for your help.

Comment (hardikshah197) [reply content/en/docs/tasks/administer-cluster/namespaces.md]: Done with the requested changes

**Review by sftim — COMMENTED** [review]
/approve

Thanks @hardikshah197 
Thanks also to @tengqm for helpful reviewing

**Pull request merged**